core/src/main/java/org/commonjava/maven/ext/core/state/RESTState.java

```diff
@@ -95,8 +105,9 @@ public void initialise( Properties userProps )
         int restRetryDuration = Integer.parseInt( userProps.getProperty( REST_RETRY_DURATION_SEC,
                                                                          String.valueOf( DefaultTranslator.RETRY_DURATION_SEC ) ) );
 
-        restEndpoint = new DefaultTranslator( restURL, restMaxSize, restMinSize, repositoryGroup, vState.getIncrementalSerialSuffix(),
-                                              restHeaders, restConnectionTimeout, restSocketTimeout, restRetryDuration );
+        restEndpoint = new DefaultTranslator( restURL, restMaxSize, restMinSize, repositoryGroup, brewPullActive, temporaryBuild,
```

Comment:
Question - if `brewPullAction` and `temporaryBuild` are replacing `repositoryGroup` why do we still have `repositoryGroup` here?

Reply:
For backwards compatibility so we can deploy new PME independently on DA/Repour. We can either use the new PME version earlier than them or later.

Reply:
Ok makes sense. Couple of options I guess - I release 4.3 now, and then this becomes 4.4 with `repositoryGroup` removed or we keep it and release in with 4.3 but mark it as deprecated via the `ConfigValue` annotation )with appropriate updates in the documentation) and then remove it in future.

Reply:
Marking as resolved - seems sensible to have a release with backwards compatibility